scripts/src/test/org/oppia/android/scripts/AccessibilityLabelCheckTest.kt

```diff
@@ -0,0 +1,195 @@
+package org.oppia.android.scripts
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+import org.oppia.android.testing.assertThrows
+import java.io.ByteArrayOutputStream
+import java.io.PrintStream
+
+class AccessibilityLabelCheckTest {
+  private val outContent: ByteArrayOutputStream = ByteArrayOutputStream()
+  private val originalOut: PrintStream = java.lang.System.out
+
+  @Rule
+  @JvmField
+  public var tempFolder: TemporaryFolder = TemporaryFolder()
+
+  @Before
+  fun setUpTests() {
+    tempFolder.newFolder("testfiles")
+    java.lang.System.setOut(PrintStream(outContent))
+  }
+
+  @After
+  fun restoreStreams() {
+    java.lang.System.setOut(originalOut)
+  }
+
+  @Test
+  fun testAccessibilityLabel_labelPresent_activitiesAreDefinedWithAccessibilityLabels() {
```

Comment:
also write positive test, that all activities have labels.

Reply:
Apologies! I didn't catch this correctly. This test actually does the same thing which you suggested. Here all the activities are defined with the label, and we are asserting that the script check is successful here. Please let me know if this isn't still clear.